custom-recipes/pi-system-retrieve-list/recipe.json

```diff
@@ -229,7 +229,7 @@
         {
             "name": "max_count",
             "label": "Max count",
-            "visibilityCondition": "['RecordedData'].includes(model.data_type)",
+            "visibilityCondition": "false && ['RecordedData'].includes(model.data_type)",
```

Comment:
Yes, this maxCount is now just confusing to the user since it's now acts as a batch size rather than pi-system's maxCount. Its values does not really impact the performance, so no real value to make it visible.